pkg/crypto/webkms/remotecrypto.go

```diff
@@ -827,3 +828,54 @@ func closeResponseBody(respBody io.Closer, logger spi.Logger, action string) {
 		logger.Errorf("Failed to close response body for '%s' REST call: %s", action, err.Error())
 	}
 }
+
+// Issuer
+
+// CLGetCredentialDefinition - To be implemented.
+func (r *RemoteCrypto) CLGetCredentialDefinition(kh interface{}) (*cl.CredentialDefinition, error) {
```

Comment:
It would be surprising that a WebKMS would be implementing functionality at the level of credential definitions, credentials, etc.

Suggest that the Crypto interface should remain about Crypto and not about credentials.